worlds/factorio/data/mod_template/control.lua

```diff
@@ -134,40 +134,96 @@ end
 
 script.on_event(defines.events.on_player_changed_position, on_player_changed_position)
 {% endif %}
-
+function count_energy_bridges()
+    local count = 0
+    for i, bridge in pairs(storage.energy_link_bridges) do
+        if validate_energy_link_bridge(i, bridge) then
+            count = count + 1 + (bridge.quality.level * 0.3)
+        end
+    end
+    return count
+end
+function get_energy_increment(bridge)
+	return ENERGY_INCREMENT + (ENERGY_INCREMENT * 0.3 * bridge.quality.level)
+end
 function on_check_energy_link(event)
     --- assuming 1 MJ increment and 5MJ battery:
     --- first 2 MJ request fill, last 2 MJ push energy, middle 1 MJ does nothing
     if event.tick % 60 == 30 then
-        local surface = game.get_surface(1)
         local force = "player"
-        local bridges = surface.find_entities_filtered({name="ap-energy-bridge", force=force})
-        local bridgecount = table_size(bridges)
-        storage.forcedata[force].energy_bridges = bridgecount
+        local bridges = storage.energy_link_bridges
+		local bridgecount = count_energy_bridges()
+		storage.forcedata[force].energy_bridges = bridgecount
```

Comment:
```suggestion
		local bridgecount = count_energy_bridges()
		storage.forcedata[force].energy_bridges = bridgecount
```

indentation broke again

Reply:
Fixed the indentation locally.